src/sbml/packages/render/sbml/DefaultValues.cpp

```diff
@@ -3777,7 +3777,7 @@ LIBSBML_EXTERN
 char *
 DefaultValues_getFillRuleAsString(const DefaultValues_t * dv)
 {
-  return (char *)(dv->getFillRuleAsString()).c_str();
+  return (char *)(FillRule_toString((FillRule_t)(dv->getFillRule())));
```

Comment:
@skeating this is still invalid. Either it would have to be returned as `const char*`, or we need a `strdup`, as the resulting ptr is not owned by the caller otherwise.